automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeDataprocSpec.scala

```diff
@@ -87,27 +90,44 @@ class RuntimeDataprocSpec
   "should stop/start a Dataproc cluster with workers and preemptible workers" in { project =>
     val runtimeName = randomClusterName
 
-    // 2 workers and 5 preemptible workers
-    val createRuntimeRequest = defaultCreateRuntime2Request.copy(
-      runtimeConfig = Some(
-        RuntimeConfigRequest.DataprocConfig(
-          Some(2),
-          Some(MachineTypeName("n1-standard-4")),
-          Some(DiskSize(100)),
-          Some(MachineTypeName("n1-standard-4")),
-          Some(DiskSize(100)),
-          None,
-          Some(5),
-          Map.empty
-        )
-      ),
-      toolDockerImage = Some(ContainerImage(LeonardoConfig.Leonardo.hailImageUrl, ContainerRegistry.GCR))
-    )
-
     val res = dependencies.use { dep =>
       implicit val client = dep.httpClient
+      val bucketName = GcsBucketName("leo-test-bucket")
+      val startScriptObjectName = GcsBlobName("test-script.sh")
       for {
+        // Set up test bucket for startup script
+        _ <- google2StorageResource.use { storage =>
+          for {
+            _ <- storage.insertBucket(project, bucketName).compile.drain
```

Comment:
Noticed error:
```
message: Failed to create runtime gpalloc-qa-master-g8gsgqt/automation-test-aojtyhzhz, status: 403 Forbidden body: {"causes":[],"exceptionClass":"org.broadinstitute.dsde.workbench.leonardo.http.service.BucketObjectAccessException","message":"ron.weasley@quality.firecloud.org does not have access to gs://leo-test-bucket/test-script.sh","source":"leonardo","stackTrace":[],"statusCode":403}
```

You may need to grant permission to the bucket object to Ron's pet

Reply:
ah yea...I saw that in old code, and just figured I'd try it without granting explicitly first :)